docs/csharp/language-reference/builtin-types/integral-numeric-types.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Integral numeric types - C# reference"
+description: "Learn the range, storage size, and uses for each of the integral numeric types."
+ms.date: 06/25/2019
+f1_keywords:
+  - "byte"
+  - "byte_CSharpKeyword"
+  - "sbyte_CSharpKeyword"
+  - "sbyte"
+  - "short"
+  - "short_CSharpKeyword"
+  - "ushort"
+  - "ushort_CSharpKeyword"
+  - "int_CSharpKeyword"
+  - "int"
+  - "uint"
+  - "uint_CSharpKeyword"
+  - "long_CSharpKeyword"
+  - "long"
+  - "ulong_CSharpKeyword"
+  - "ulong"
+helpviewer_keywords: 
+  - "integral types, C#"
+  - "Visual C#, integral types"
+  - "types [C#], integral types"
+  - "ranges of integral types [C#]"
+  - "byte keyword [C#]"
+  - "sbyte keyword [C#]"
+  - "short keyword [C#]"
+  - "ushort keyword [C#]"
+  - "int keyword [C#]"
+  - "uint keyword [C#]"
+  - "long keyword [C#]"
+  - "ulong keyword [C#]"
+---
+# Integral numeric types  (C# reference)
+
+The **integral numeric types** are a subset of the **simple types** and can be initialized with [*literals*](#integral-literals). All integral types are also value types.
+
+All integral numeric types support [arithmetic](../operators/arithmetic-operators.md), [bitwise logical](../operators/bitwise-and-shift-operators.md), [comparison, and equality](../operators/equality-operators.md) operators.
+
+## Sizes and ranges
+
+The following table shows the sizes and ranges of the integral types:
+
+|Type|Range|Size|  
+|----------|-----------|----------|  
+|`sbyte`|-128 to 127|Signed 8-bit integer|  
+|`byte`|0 to 255|Unsigned 8-bit integer|  
+|`short`|-32,768 to 32,767|Signed 16-bit integer|  
+|`ushort`|0 to 65,535|Unsigned 16-bit integer|  
+|`int`|-2,147,483,648 to 2,147,483,647|Signed 32-bit integer|  
+|`uint`|0 to 4,294,967,295|Unsigned 32-bit integer|  
+|`long`|-9,223,372,036,854,775,808 to 9,223,372,036,854,775,807|Signed 64-bit integer|  
+|`ulong`|0 to 18,446,744,073,709,551,615|Unsigned 64-bit integer|  
+
+The default value for all integral types is `0`. Each of the integral types has constants named `MinValue` and `MaxValue` for the minimum and maximum value for that type.
+
+Use the <xref:System.Numerics.BigInteger?displayProperty=nameWithType> structure to represent an arbitrarily large signed integer.
+
+## Integral literals
+
+Integral literals can be specified as *decimal literals*, *hexadecimal literals*, or *binary literals*. An example of each is shown below:
+
+```csharp
+var decimalLiteral = 42;
+var hexLiteral = 0x2A;
+var binaryLiteral = 0b_0010_1010;
+```
+
+Decimal literals don't require any prefix. The `x` or `X` prefix signifies a *hexadecimal literal*. The `b` or `B` prefix signifies a *binary literal*. The declaration of `binaryLiteral` demonstrates the use of `_` as a digit separator. Binary literals and the digit separator `_` are supported starting with C# 7.0.
+
+## Type of an integral literal
+
+If an integral literal has no suffix, its type is the first of the following types in which its value can be represented:
```

Comment:
suffixes haven't been discussed. Perhaps a separate section listing them right before this section?